src/aap_eda/tasks/ruleset.py

```diff
@@ -38,10 +39,6 @@ def activate_rulesets(is_restart: bool, activation_id: int) -> None:
         logger.info(f"Activation id: {activation_id} is deleted")
         return
 
-    # TODO(hsong): move this to views
-    if is_restart:
```

Comment:
@hsong-rh If the scheduler does a restart because the activation is unresponsive how would this increment work if its in the view?

Reply:
As I known, the restart counter is only increased by manually clicking the restart button

Reply:
I don't think so, it is the contrary, the counter is increased for every automatic restart. It is a way to indicate to the user that something could go wrong.

Reply:
More context: https://github.com/ansible/eda-qa/blob/main/eda_qa/tests/api/test_activations.py#L286

Reply:
@Alex-Izquierdo https://github.com/ansible/eda-server/blob/main/src/aap_eda/api/views/activation.py#L340 so this increasing counter is not needed?

Reply:
@hsong-rh I think that is fine (has been there for five months). What I say that the counter should be increased as well with automatic restarts.

Reply:
@Alex-Izquierdo I remembered we only increase the counter in one side. Anyway I will move back the restart counter, thanks!